google-cloud-bigtable/Rakefile

```diff
@@ -122,7 +122,7 @@ end
 
 namespace :ci do
   desc "Run the CI build, with acceptance tests."
-  task :acceptance do
+  task :smoke_test do
```

Comment:
The build needs an `acceptance` task to exist. Why is this renamed?

Reply:
This is a mistake. I missed a replacement when updating the code generator templates. (This PR is not ready for review yet.)

Reply:
Gotcha. Understood on WIP. I just like to be familiar with upcoming changes and watch the progress. Looks great so far!